### PR TITLE
Investigator Jacket Fixes

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR removes the unused investigator jacket and repaths the detective jacket (which is actually used as the investigator jacket) to investigator.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!bugfix

**Comment by SleepyGemmy** [comment]
!review

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -141,28 +141,34 @@ obj/item/clothing/suit/apron/overalls/blue
...
 	item_state = "hos_jacket"
 
-//Detective
-
-/obj/item/clothing/suit/storage/det_jacket
-	name = "detective's jacket"
-	desc = "Stylish yet comfortable professional jacket manufactured by CL corporation for detectives."
+/obj/item/clothing/suit/storage/security/investigator
+	name = "investigator jacket"
+	desc = "An investigator jacket. Stylish, professional, yet comfortable."
```
Comment (alsoandanswer) [inline code/modules/clothing/suits/jobs.dm:146]: try not to put changes into bugfixes, thanks

Comment (SleepyGemmy) [reply code/modules/clothing/suits/jobs.dm:146]: not really a change if the jacket has been used for the investigators before.

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**